scapy/utils.py

```diff
@@ -1904,3 +1905,31 @@ def whois(ip_address):
         else:
             break
     return b"\n".join(lines[3:])
+
+#######################
+#   PERIODIC SENDER   #
+#######################
+
+
+class PeriodicSenderThread(threading.Thread):
```

Comment:
This does not seem to be used anywhere else. If so, it must be removed.

Reply:
I will remove it